### PR TITLE
cronie: init at 1.5.5

### PR DESCRIPTION
**Pull request by marsam**
###### Motivation for this change

Add https://github.com/cronie-crond/cronie

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by aanderse** [comment]
No module additions or changes. Out of curiosity how are you using this?

**Comment by marsam** [comment]
I just found it exists, and seems to be well maintained by the people of Fedora. 
I'm going to create another PR, in a day or two, to make cronie to be used by the cron module.

**Comment by aanderse** [comment]
I wondered if that was where this was going :+1:

File: pkgs/tools/system/cronie/default.nix [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+stdenv.mkDerivation rec {
+  pname = "cronie";
+  version = "1.5.5";
+
+  src = fetchurl {
+    url = "https://github.com/cronie-crond/cronie/releases/download/cronie-${version}/cronie-${version}.tar.gz";
+    sha256 = "1b0ffbjxmpf0yyfrympfc2qidzsmk55p8m0q50il6m750nawfd5y";
+  };
+
+  meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/tools/system/cronie/default.nix:12]:
```suggestion
  meta = with lib; {
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**